autogpt_platform/frontend/src/components/agptui/composite/LibraryActionSubHeader.tsx

```diff
@@ -0,0 +1,22 @@
+"use client";
```

Comment:
does *this* need to be a client component

Reply:
yes because it uses a context which is a client-side only feature